addon/fonts.py

```diff
@@ -23,6 +23,14 @@ def get_col_name(idx):
 def get_col_path(idx):
     return util.col_media_path(get_col_name(idx))
 
+def assure_primitive_col_media():
+    primitive_dir = util.addon_path("primitives")
+    primitive_files = os.listdir(primitive_dir)
+    for f in primitive_files:
+        src_path = util.addon_path("primitives", f)
+        dest_path =  util.col_media_path('_primitive_' + f)
+        shutil.copy(src_path, dest_path)
```

Comment:
I currently get this error on the latest version:

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue to the add-on author.
Debug info:
Anki 2.1.66 (70506aeb) Python 3.9.15 Qt 6.5.0 PyQt 6.5.0
Platform: Linux-6.4.12-zen1-1-zen-x86_64-with-glibc2.35
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2023-09-01 12:04:11
Add-ons possibly involved: ⁨migaku_kanji_addon⁩

Caught exception:
Traceback (most recent call last):
  File "aqt.progress", line 118, in handler
  File "aqt.main", line 217, in on_window_init
  File "aqt.main", line 316, in setupProfile
  File "aqt.main", line 495, in loadProfile
  File "_aqt.hooks", line 3977, in __call__
  File "/home/julius/.var/app/net.ankiweb.Anki/data/Anki2/addons21/migaku_kanji_addon/__init__.py", line 126, in on_profile_open
    CardType.upsert_all_models()
  File "/home/julius/.var/app/net.ankiweb.Anki/data/Anki2/addons21/migaku_kanji_addon/card_type.py", line 207, in upsert_all_models
    cls.assure_global_col_media()
  File "/home/julius/.var/app/net.ankiweb.Anki/data/Anki2/addons21/migaku_kanji_addon/card_type.py", line 198, in assure_global_col_media
    fonts.assure_col_media()
  File "/home/julius/.var/app/net.ankiweb.Anki/data/Anki2/addons21/migaku_kanji_addon/fonts.py", line 120, in assure_col_media
    assure_primitive_col_media()
  File "/home/julius/.var/app/net.ankiweb.Anki/data/Anki2/addons21/migaku_kanji_addon/fonts.py", line 32, in assure_primitive_col_media
    shutil.copy(src_path, dest_path)
  File "shutil", line 427, in copy
  File "shutil", line 264, in copyfile
IsADirectoryError: [Errno 21] Is a directory: '/home/julius/.var/app/net.ankiweb.Anki/data/Anki2/addons21/migaku_kanji_addon/primitives/cursive'
```

Reply:
Ah, sorry, the cursive type images in the subdirectory that weren't going to be used yet in this version but I left them there for the next PR/commit.  Here's the patch that actually uses them and copies the subdirectory contents correctly.